packages/teleport-component-generator-vue/__tests__/integration/component-with-conditional-root-array-child-node.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://raw.githubusercontent.com/teleporthq/uidl-definitions/master/component.json",
+    "name": "ComponentWithConditionalArrayNode",
```

Comment:
I think we should stop adding json files inside the tests. There is an issue to completly eliminate them.
Instead, you can use the functions inside @teleporthq/teleport-shared/lib/builders/uidl-builders.
There are also examples of conditional components in `packages/teleport-component-generator-react/__tests__/integration/component-conditional.ts` or in `packages/teleport-component-generator-vue/__tests__/integration/component-conditional.ts`